tools/js-sdk-release-tools/src/changelog/changelogGenerator.ts

```diff
@@ -23,6 +36,14 @@ export interface ChangelogItem {
 }
 
 export class Changelog {
+    // TODO: refactor: use unified changelog item,
```

Comment:
Is this todo still valid?

Reply:
yes, but I don't plan to refactor in this pr, since it's a large change, it will make review experience really bad.